pages/design/composed-template-empty.md

```diff
@@ -0,0 +1,78 @@
+---
+name:
+description: |
+  This rule checks ...
+
```

Comment:
Should add `type: atomic|composed`

Reply:
Believe the agreement on the last call, was we will be able to decipher if a rule is atomic or composed, based on listing of atomic rules in the front-matter, so this might be not necessary.